prow/config.gen.yaml

```diff
@@ -441,6 +441,51 @@ postsubmits:
             cpu: "1"
             memory: 1Gi
 
+  maistra/istio:
+  - name: istio-update-operator
+    decorate: true
+    path_alias: maistra.io/istio
+    skip_report: false
+    branches:
+      - ^maistra-2.1$
+      # Allow for testing
+      - playground
+    labels:
+      preset-github: "true"
+    extra_refs:
+    - base_ref: main
+      org: maistra
+      path_alias: maistra.io/test-infra
+      repo: test-infra
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - ../test-infra/tools/automator.sh
+        - -o maistra
+        - -r istio-operator
+        - -f /creds-github/github-token
+        - -c make clean gen restore-manifest-dates
```

Comment:
We probably want to limit this to: `make clean update-2.1-charts` for now.  Updating the charts from other releases would require updating the unit tests to match the version used in the release (e.g. 2.0.5.1).  You can look at some of the other chart update PRs for an example.  Maybe it's worth having a separate job for each version, e.g. for 2.0 maybe, `make clean update-2.0-charts update-2.0-templates update-2.0-expected-version` (that last target doesn't exist, yet)`

Reply:
It may, but I think that's out of scope.  2.0 charts/templates come from istio-operator@maistra-2.0, similarly for 1.0 and 1.1 charts.  This is why I was suggesting separate jobs.  That said, I get your point: if all of the updates are made in a short period of time, the gen-check will fail for all of them.  Maybe it's best to just let those other PRs get stuck and require manual intervention.  Maybe just two different automation jobs: one to keep the release charts up to date, and one to keep the dev charts up to date.  Or maybe one for the dev charts and one for all charts?  Keep in mind, for the other charts, we need to bump the replace version (for 2.0 in 2.1 branch, which should be formatted \d.\d.\d) and the default profile unit tests (for 1.x and 2.0, and should match the version in the charts).